nixos/modules/services/torrent/deluge.nix

```diff
@@ -20,45 +44,143 @@ in {
             Number of files to allow deluged to open.
           '';
         };
+
+        config = mkOption {
+          type = types.attrs;
+          example = literalExample ''
+            {
+              download_location = "/srv/torrents/";
+              max_upload_speed = "1000.0";
+              share_ratio_limit = "2.0";
+              allow_remote = true;
+              daemon_port = 58846;
+              listen_ports = [ 6881 6889 ];
+              }
+          '';
+          description = ''
+            Deluge core configuration for the core.conf file. Only has
+            an effect when <option>services.deluge.declarative</option>
+            is set to <literal>true</literal>.
+            String values must be quoted, integer and boolean values must not.
```

Comment:
Oh I found something we can link to: https://git.deluge-torrent.org/deluge/tree/deluge/core/preferencesmanager.py#n41

There's all options + the defaults there.